src/ui/server/content-strings/pages/index.ts

```diff
@@ -126,4 +130,5 @@ export const PAGES = {
   PROBLEM_WITH_SERVICE_PAGE,
   QUOTE: QUOTE_PAGES,
   INSURANCE: INSURANCE_PAGES,
+  YOUR_BUYER: YOUR_BUYER_DETAILS_PAGE,
```

Comment:
Can we move this into insurance please and export like the other insurance sections, e.g

```js
import * as EXPORTER_BUSINESS from './exporter-business';
```